ui/ui-tests/__api_recordings__/LiteratureDetail_3323728427/recording.har

```diff
@@ -478,123 +493,27 @@
               "name": "connection",
               "value": "close"
             },
-            {
-              "name": "content-length",
-              "value": "70"
-            }
-          ],
-          "headersSize": 308,
-          "httpVersion": "HTTP/1.1",
-          "redirectURL": "",
-          "status": 200,
-          "statusText": "OK"
-        },
-        "startedDateTime": "2019-11-08T13:07:24.699Z",
-        "time": 137,
-        "timings": {
-          "blocked": -1,
-          "connect": -1,
-          "dns": -1,
-          "receive": 0,
-          "send": 0,
-          "ssl": -1,
-          "wait": 137
-        }
-      },
-      {
-        "_id": "dfdf8cc1967898402fa0405eb59c2694",
-        "_order": 0,
-        "cache": {},
-        "request": {
-          "bodySize": 0,
-          "cookies": [],
-          "headers": [
-            {
-              "name": "accept",
-              "value": "application/json, text/plain, */*"
-            },
-            {
-              "name": "x-devtools-emulate-network-conditions-client-id",
-              "value": "7003F9BBBD134E6B975A6A160E20BF6C"
-            },
-            {
-              "name": "user-agent",
-              "value": "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) HeadlessChrome/67.0.3391.0 Safari/537.36"
-            },
-            {
-              "name": "referer",
-              "value": "http://localhost:8080/literature/1472986"
-            }
-          ],
-          "headersSize": 356,
-          "httpVersion": "HTTP/1.1",
-          "method": "GET",
-          "queryString": [],
-          "url": "http://localhost:8080/api/accounts/me"
-        },
-        "response": {
-          "bodySize": 258,
-          "content": {
-            "mimeType": "application/json",
-            "size": 258,
-            "text": "{\n  \"message\": \"The server could not verify that you are authorized to access the URL requested. You either supplied the wrong credentials (e.g. a bad password), or your browser doesn't understand how to supply the credentials required.\", \n  \"status\": 401\n}\n"
```

Comment:
The ```401``` you mentioned above was there from before as well. It's from the ```api/accounts/me``` endpoint because we aren't logged in